.github/workflows/CI.yaml

```
@@ -39,12 +39,13 @@ jobs:
 
   check:
     # needs: spec_version
+    needs: check-commit-message
     runs-on: [self-hosted, epyc-4, k8s-runner]
     env:
       RUSTUP_HOME: /tmp/rustup_home
     steps:
       - name: Cancel Previous Runs
-        if: ${{ github.event_name == 'pull_request' }}
+        if: ${{ github.event_name == 'pull_request'}}
```

I like the previous one.
```suggestion
        if: ${{ github.event_name == 'pull_request' }}
```